src/AutoRest.CSharp/Common/Output/Models/Types/EnumType.cs

```diff
@@ -38,6 +38,7 @@ public EnumType(InputEnumType input, string defaultNamespace, string defaultAcce
 
             DefaultName = input.Name.ToCleanName();
             DefaultAccessibility = input.Accessibility ?? defaultAccessibility;
+            IsAccessibilityOverridden = input.Accessibility != null;
```

Comment:
Why do we need this field in output model?  Output model will contain finally information and will not need any computation.  Can we just set the accessibility value?

Reply:
This property is used to send to post processor. If a model is access overridden, the post process should not remove it.